src/pages/money/money.tsx

```
@@ -245,13 +327,21 @@ const Money = () => {
-      localStorage.setItem('dailyEntries', JSON.stringify(next));
-      if (!next.length) setDeleteMode(false);
-      return next;
-    });
+  const deleteEntry = async (id: number) => {
+    try {
+      const res = await useDailyDeleteMutation(id);
+      if (res.isSuccess) {
+        alert('삭제 되었습니다.');
+        window.location.reload();
+        setEntries((prev) => {
+          const next = prev.filter((e) => e.id !== id);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)